token-lending/program/tests/deposit_reserve_liquidity.rs

```diff
@@ -16,7 +16,7 @@ async fn test_success() {
     );
 
     // limit to track compute unit increase
-    test.set_bpf_compute_max_units(30_000);
+    test.set_bpf_compute_max_units(35_000);
```

Comment:
Is there a way we can assert that the reserve interest has been refreshed in these tests?